tests/include/SimpleDataTest.h

```diff
@@ -34,8 +34,8 @@ class SimpleDataTest : public CPPUNIT_NS::TestFixture
 {
     CPPUNIT_TEST_SUITE(SimpleDataTest);
 
-    CPPUNIT_TEST(testWriteRead);
     CPPUNIT_TEST(testNullWrite);
+    CPPUNIT_TEST(testWriteRead);
```

Comment:
Yes, I want to read the data later on via h5py :)
the two tests use the same file (overwrite).

Am 10. Februar 2015 06:32:53 MEZ, schrieb f-schmitt-zih <notifications@github.com>:
>>      CPPUNIT_TEST(testNullWrite);
>> +    CPPUNIT_TEST(testWriteRead);
>
>Any reason for that change? :) I liked my order
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ComputationalRadiationPhysics/libSplash/pull/153/files#r24390040